promotion/datastore.go

```diff
@@ -150,6 +152,23 @@ func NewPostgres(databaseURL string, performMigration bool) (*Postgres, error) {
 		return nil, err
 	}
 
+	// setup instrumentation using sqlstats
+	// Create a new collector, the name will be used as a label on the metrics
+	collector := sqlstats.NewStatsCollector("db", db)
+
+	// Register it with Prometheus
+	prometheus.MustRegister(collector)
+
+	if env := os.Getenv("ENV"); env == "production" {
```

Comment:
I'm relatively new to Go but it might be simpler to do the following 😄 
```suggestion
	if os.Getenv("ENV") == "production" {
```

Reply:
:+1: